### PR TITLE
Update Docker to v20.10.8

### PR DESCRIPTION
**Pull request by hakman**
https://github.com/moby/moby/releases/tag/v20.10.8

### CODE REVIEW

File: upup/pkg/fi/cloudup/containerd.go [hunk 1]
```diff
@@ -207,6 +207,7 @@ func findAllContainerdDockerMappings() map[string]string {
 		"1.4.3": "20.10.0",
 		"1.4.4": "20.10.6",
 		"1.4.6": "20.10.7",
+		"1.4.9": "20.10.8",
```
Comment (johngmyers) [inline upup/pkg/fi/cloudup/containerd.go:210]: Should we change the value of `containerdFallbackVersion`?

**Review by johngmyers — APPROVED** [review]

File: upup/pkg/fi/cloudup/containerd.go (hunk 1 above)
Comment (hakman) [reply upup/pkg/fi/cloudup/containerd.go:210]: I want to cherry-pick this to 1.21 without changing the fallback.

**Pull request merged**